corehq/apps/reports/models.py

```diff
@@ -758,31 +765,38 @@ def send(self):
             for language, emails in self.recipients_by_language.items():
                 self._get_and_send_report(language, emails)
 
+    def _get_or_create_uuid(self):
+        if not self.uuid:
+            self.uuid = random_hex()
```

Comment:
Just noticed that this is `django_otp.utils.random_hex`. Maybe rather use `uuid.uuid4().hex`